retail-shopping/camera-imx219/setup_imx219-1640x1232.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
```

Comment:
Hi @reesegrimsley 

I suggest you modify the **scripts/setup_camera.sh** script it self.
since this is not deviated much from the original script